scripts/pi-hole/js/groups-adlists.js

```diff
@@ -30,6 +30,83 @@ $(function () {
   getGroups();
 });
 
+function format(data) {
+  // Generate human-friendly status string
+  var statusText = "Unknown";
+  var numbers = true;
+  if (data.status !== null) {
+    switch (parseInt(data.status, 10)) {
+      case 0:
+        statusText =
+          data.enabled === 0
+            ? "List is disabled and not checked"
+            : "List was not downloaded so far";
+        numbers = false;
+        break;
+      case 1:
+        statusText = 'List download was successful (<span class="list-status-1">OK</span>)';
+        break;
+      case 2:
+        statusText =
+          'List unchanged upstream, Pi-hole used a local copy (<span class="list-status-2">OK</span>)';
+        break;
+      case 3:
+        statusText =
+          'List unavailable, Pi-hole used a local copy (<span class="list-status-3">check list</span>)';
+        break;
+      case 4:
+        statusText =
+          'List unavailable, there is no local copy of this list available on your Pi-hole (<span class="list-status-4">replace list</span>)';
+        numbers = false;
+        break;
+
+      default:
+        statusText =
+          'Unknown (<span class="list-status-0">' + parseInt(data.status, 10) + "</span>)";
+        break;
+    }
+  }
+
+  var invalidStyle =
+    data.invalid_domains !== null && data.invalid_domains > 0 && numbers === true
+      ? ' style="color:red; font-weight:bold;"'
+      : "";
+
+  // Compile extra info for displaying
+  return (
+    "<table>" +
+    '<tr class="dataTables-child"><td>Health status of this list:</td><td>' +
+    statusText +
+    '</td></tr><tr class="dataTables-child"><td>This list was added to Pi-hole&nbsp;&nbsp;</td><td>' +
+    utils.datetimeRelative(data.date_added) +
+    "&nbsp;(" +
+    utils.datetime(data.date_added, false) +
+    ')</td></tr><tr class="dataTables-child"><td>Database entry was last modified&nbsp;&nbsp;</td><td>' +
+    utils.datetimeRelative(data.date_modified) +
+    "&nbsp;(" +
+    utils.datetime(data.date_modified, false) +
+    ')</td></tr><tr class="dataTables-child"><td>The list contents were last updated&nbsp;&nbsp;</td><td>' +
+    (data.date_updated !== null
```

Comment:
```suggestion
    (data.date_updated !== 0
```
should avoid the 1970 date.